build/run-tests.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-docker-compose build
+docker-compose build --no-cache
```

Comment:
This looks like it ideally belongs in a separate commit?

Reply:
Or removed? Why do we want to prevent use of the cache?

Reply:
When I was building, it was using a cached version of the layer that does the `git clone`, so I was never pulling the latest version of the database migrations.

Probably need to investigate why, but in the short term, this was the fix.

Reply:
A good explanation of how layer caching works and why I was seeing the problem here: https://thenewstack.io/understanding-the-docker-cache-for-faster-builds/

Reply:
😞 I thought Docker was more intelligent than that... 😢